app/controllers/api/v1/keywords_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class KeywordsController < ApplicationController
+      include ::Pagy::Backend
+      include API::V1::Pagy::JSONAPIConcern
+      include ErrorHandlerConcern
+
+      def index
+        pagy, keywords_list = pagy(keywords)
+
+        if keywords.any?
```

Comment:
what happens when you pass empty keywords_list, is the empty case really needed?

Reply:
It's not changing much, the idea was to have a custom message.

Both are perfectly well working. 
I'm ok to remove it as the pros/cons are very small.

With this custom message:
- Lighter result
- Custom message can be reused by API consumer
```json
{
    "meta": "You do not have any keyword yet.",
    "data": []
}
```

Without this message:
- Format is more consistent between empty/not_empty queries
- No custom message
```json
{
    "data": [],
    "meta": {
        "total_pages": 1
    },
    "links": {
        "first": "http://localhost:3000/api/v1/keywords?page=1",
        "self": "http://localhost:3000/api/v1/keywords?page=1",
        "next": null,
        "prev": null,
        "last": "http://localhost:3000/api/v1/keywords?page=1"
    }
}
```

What do you think, @ankitkalia1195 ? :)

Reply:
@junan @malparty I am fine with both, but such a logic is usually handled on client end, where they check size of data returned. What do you think @malparty

Reply:
@ankitkalia1195 @malparty I just checked our previous ([Braive](https://github.com/nimblehq/braive-web/blob/development/app/controllers/api/v1/journals_controller.rb#L10), [The1-API](https://github.com/nimblehq/the1-api/blob/development/app/controllers/v2/districts_controller.rb#L6)) projects, we did not handle empty cases and let the client handle it as just Ankit said.

Both ways are fine to me but I am preferring more not to handle the empty case and let the client handle it we already followed it in our projects, it will be better to follow the same for consistency.

Reply:
After second reading, yes, I think the call for simplicity wins at the end. I will remove this case ;-)

Reply:
Okay, done! ✅